operations/helm/charts/grafana-agent/CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ internal API changes are not present.
 
 Unreleased
```

Comment:
Looks like we'll have to release this immediately, can you add a section for 0.2.0?

We'll probably disable requiring version bumps for every change after this gets merged.

Reply:
Will do